GraphQL/Inputs/CreateExportInput.graphql

```diff
@@ -1,16 +1,55 @@
+"""
+The provided input arguments.
+"""
 input ArgsInput {
+    """
+    An array of company ids included in the export.
+    """
     companies: [Int!]!,
+    """
+    The start date of the export.
+    """
     dateFrom: Date
+    """
+    The end date of the export.
+    """
     dateTo: Date
 }
 
+"""
+The input for creating an export.
+"""
 input CreateExportInput {
+    """
+    The user id to create an export for.
+    """
     userId: Int!
+    """
+    The impersonator that is creating the export.
+    """
     impersonatorId: Int
+    """
+    The account to create an export for.
+    """
     accountId: Int!
+    """
+    The account type to create an export for.
+    """
     accountType: String!
+    """
+    The type of export to create.
+    """
     type: String!
+    """
+    The type of export to create.
+    """
```

Comment:
The description on these two are identical

`generatorType` what it generator to use, e.g. format. Currently csv is the only supported `generator`.  and type defines what processor to use, which is basically what to export. E.g. `bill`, `job` etc... 

At some point the package should automatically generate enums for these instead.

Reply:
Yeah, would be good to generate enums. Should I add manually-created enums to this for now?

Reply:
Let's wait with that, as it will break frontend that does not insert the data as enums. 

Also because type is being extended in the platform. 👍